src/Modules/Station/Components/OnlineUsers/OnlineUsers.tsx

```diff
@@ -145,9 +145,7 @@ export class OnlineUsersComponent extends BaseComponent<IProps, IState> {
           </div>
         ))}
         {anotherUsersCount > 0 ? (
-          <span className="align-text-left">
-            and {anotherUsersCount} more
-          </span>
+          <span className="align-text-left">and {anotherUsersCount} more</span>
```

Comment:
Is it really needed to change this?

Reply:
It didn't change anything, just some break line